### ISSUE TITLE
Mi Band 7 ( NO NFC _ NO PRO )

### ISSUE THREAD

**Issue by c7274864** [theengs/decoder#422]
Bonjour

J'ai testé TheengsGateway sous HomeAssistant en Addons
malgré une recherche, je ne trouve pas de réponses
https://github.com/theengs/decoder/issues?q=mi+band

Certaines informations remontent , Présence / Rssi / Fréquence cardiaque ..

Mais les pas remonte inconnu, et MAC est aussi en inconnu.

Une évolution est possible pour avoir cette info et d'autre comme la surveillance du sommeil dans un futur ?.

Pour résumer:
******************************
Good morning

I tested TheengsGateway under HomeAssistant in Addons
despite searching, I can't find any answers.
https://github.com/theengs/decoder/issues?q=mi+band

Some information comes back, Presence / Rssi / Heart rate..

But the steps go back unknown, and MAC is also unknown.

Is an evolution possible to have this information and others such as sleep monitoring in the future?

To sum up:
![Capture d'écran 2023-10-02 205104](https://github.com/theengs/decoder/assets/143031506/37d52f84-106d-46a8-9c15-9c11293e1003)

Thx

**Comment by DigiH** [comment]
Hi @c7274864 

Please also have a look at this thread

https://github.com/1technophile/OpenMQTTGateway/issues/1642

It seems that Mi Band 7 does not broadcast servicedata with the steps included any longer, as the Mi Bands 4-6 did. Unless there will be a firmware update to restore this, the steps won't be available for Mi Band 7.

We are looking into the Unknown for the MAC address and Device fields, but as they are mainly for background functionality like Presence Tracking it might be decided to drop them from the UI completely.

As for any other information from the Mi Bands, like sleep monitoring etc., they are not available at all though the freely broadcast advertising data, and are therefore out of the scope of Decoder.

**Comment by DigiH** [comment]
Closing, as this seems to be the status quo for the Mi Band 7.

Feel free to reopen if there is more and different information available.
